docs/shared-folder.md

```diff
@@ -0,0 +1,36 @@
+# Shared Folder
+
+In HUIT OnDemand, course staff have access to a shared folder for use in their course. Using this shared folder requires use of the terminal, whether through the OnDemand dashboard or through the terminal in the VS Code interface.
+
+The shared folder will be located at `/shared/courseSharedFolders/$CANVAS_COURSE_ID`, where `$CANVAS_COURSE_ID` can be found in your course's Canvas url: `https://canvas.harvard.edu/courses/CANVAS_COURSE_ID`.
+
+The shared folder should contain two utility scripts; one script configures a link to the folder in the home directory of whoever runs the script, and the other ensures that all content in the folder that the current user can edit has appropriate permisssions.
+
+## Creating a link to the shared folder
+
+You can create a link to the shared folder in your home folder with the included script, like so:
+
+```bash
+/shared/courseSharedFolders/$CANVAS_COURSE_ID/map-this.sh
+```
+
+That script creates a symbolic link in your home directory to the shared folder called `share`, making it accessible in interactive apps like the VS Code app, and making it easier to access than referencing its absolute location.
+
+If you prefer, you can make your own link by modifying this line:
+
+```bash
+ln -s /shared/courseSharedFolders/$COURSE_ID ~/share
+```
+
+## Fixing permissions
```

Comment:
While it's self-evident to us why permissions might need to be fixed, should we include a line that motivates this section? If we enable setgid on the shared folder, that could be factored into the explanation of when/why one would need/want to fix permissions on files in the shared folder.

Reply:
Good point. Also, in writing that, I've realized that the script should also set global read access, in case that isn't there.

Reply:
Added that line to the script: https://github.com/HUIT-Cloud-Architecture/ood/pull/15